src/App.jsx

```diff
@@ -3,26 +3,44 @@ import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { getTodos } from './api';
 
 class App extends React.Component {
   state = {
     todos: [],
     selectedUserId: 0,
   };
 
+  componentDidMount() {
+    getTodos()
+      .then(todos => (
+        this.setState({ todos: todos.data })
+      ));
+  }
+
+  clearUser = (userId) => {
+    this.setState({ selectedUserId: userId });
+  }
```

Comment:
```suggestion
  clearUser = () => {
    this.setState({ selectedUserId: 0 });
  }
```
You can just reset this value to it's initial (0), and later just call it like 
```
onClick={clearUser}
```